src/Store.js

```diff
@@ -370,7 +384,13 @@ class Store {
     const snapshot = await this._cache.get(this.snapshotPath)
 
     if (snapshot) {
-      const res = this._ipfs.files.catReadableStream ? await this._ipfs.files.catReadableStream(snapshot.hash) : await this._ipfs.catReadableStream(snapshot.hash)
+      const res =
+        this._ipfs.files.catReadableStream
```

Comment:
is `files` deprecated / removed in ipfs >= 0.41?

Reply:
No, but it becomes the [mutable file system api](https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/FILES.md),
and `ipfs.catReadableStream` is removed.

Reply:
This is good, very diligent. Might you have tested this against a previous version of orbit-db, in the case that this version is mistakenly bundled? That would be useful.

Reply:
https://circleci.com/gh/Xmader/orbit-db/5

Reply:
Great! Works for me!